### PR TITLE
open-cluster-management/backlog#3916 serviceability/doc collab

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -27,3 +27,11 @@ Section:
 *
 
 ## What are the suggested changes?
+
+The doc team and dev team works together to ensure the following standards are implemented in every task or concept:
+
+1. **Quality introductions** (formally known as Short Descriptions) that briefly discuss the component or feature and why the user should continue to read or complete the task. Style guides recommend no more than 50-word introductions.
```
Comment (dockerymick) [inline .github/ISSUE_TEMPLATE.md]: comma after feature

I need to brush up on my comma skills lol, so i'll let you be the judge of adding it lol. I am pretty sure it should be added tho lolol

**Review by dockerymick — APPROVED** [review]
I made a one comment, it's a very small change. This looks good tho! Thank you again for accepting this task

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (KevinFCormier) [inline .github/ISSUE_TEMPLATE.md]: Was "formally" meant to be "formerly"?

Comment (swopebe) [reply .github/ISSUE_TEMPLATE.md]: no you are right!

Comment (swopebe) [reply .github/ISSUE_TEMPLATE.md]: @KevinFCormier no, that is Formally (IBM Style Manual guidance for what you would call an intro).

As in: 1.
in accordance with the rules of convention or etiquette.

Comment (swopebe) [reply .github/ISSUE_TEMPLATE.md]: We follow IBM Style Manual at Red Hat, too.

**Review by dhaiducek — APPROVED** [review]
Thanks! LGTM! 👍

**Pull request merged**